search-provider/eks-search-app.c

```diff
@@ -165,6 +170,11 @@ subtree_object_info_for_interface (EksSearchApp      *self,
       info->create_type = EKS_TYPE_DISCOVERY_FEED_PROVIDER;
       info->cache = self->discovery_feed_content_providers;
     }
+  else if (g_strcmp0 (interface, "com.endlessm.Metadata") == 0)
```

Comment:
In general the name change is an improvement, but this service name might be too general. Maybe `com.endlessm.ContentMetadata` just for the DBus interface?

Reply:
That should be fine (just need to remember to update eos-companion-app-integration)